build.gradle

```diff
@@ -1897,7 +1897,6 @@ project(':clients') {
 
     testRuntimeOnly libs.jacksonDatabind
     testRuntimeOnly libs.jacksonJDK8Datatypes
-    testRuntimeOnly runtimeTestLibs
```

Comment:
`clients` needs `runtimeTestLibs` to run `CommonNameLoggingTrustManagerFactoryWrapperTest` with logging.